cartography/intel/github/teams.py

```diff
@@ -23,6 +23,18 @@
 UserRole = namedtuple('UserRole', ['user_url', 'role'])
 
 
+def backoff_handler(details: Dict) -> None:
+    """
+    Custom backoff handler for GitHub calls in this module.
+    """
+    team_name = details['kwargs'].get('team_name') or 'not present in kwargs'
+    updated_details = {**details, 'team_name': team_name}
+    logger.warning(
+        "Backing off {wait:0.1f} seconds after {tries} tries. Calling function {target} for team {team_name}"
+        .format(**updated_details),
+    )
```

Comment:
Wasn't sure how valuable this was over just using the backoff_handler over in util.py.  If you think it's not worth it, I can get rid of it.